pyartifactory/objects.py

```diff
@@ -57,6 +57,7 @@ def __init__(
         self.security = ArtifactorySecurity(self.artifactory)
         self.repositories = ArtifactoryRepository(self.artifactory)
         self.artifacts = ArtifactoryArtifact(self.artifactory)
+        self.permissions = ArtifactoryPermission(self.artifactory)
```

Comment:
OMG this is a huge oversight from the latest release, thank you for pointing it out :pray: